platforms/jvm/jvm-services/src/main/java/org/gradle/api/internal/artifacts/JavaEcosystemSupport.java

```diff
@@ -86,13 +86,13 @@ public abstract class JavaEcosystemSupport {
     @Deprecated
     public static final String DEPRECATED_JAVA_RUNTIME_RESOURCES = "java-runtime-resources";
 
-    public static void configureSchema(AttributesSchema attributesSchema, final ObjectFactory objectFactory) {
+    public static void configureSchema(AttributesSchema attributesSchema, ConfigurableAttributeDescribers attributeDescribers, final ObjectFactory objectFactory) {
```

Comment:
We can rename this

Reply:
I think splitting it would be better.  It's now doing 2 things - configuring the schema (as currently named) and configuring a collection of attribute describes outside of the schema.  There doesn't seem to be a natural connection between these two things that isn't a stretch.